iib/workers/tasks/build.py

```diff
@@ -729,10 +727,9 @@ def handle_add_request(
     with tempfile.TemporaryDirectory(prefix='iib-') as temp_dir:
         if from_index:
             with set_registry_token(overwrite_from_index_token, from_index_resolved):
-                if is_image_fbc(from_index_resolved):
-                    err_msg = 'File-Based catalog image type is not supported yet.'
-                    log.error(err_msg)
-                    raise IIBError(err_msg)
+                is_fbc = is_image_fbc(from_index_resolved)
+                if is_fbc:
+                    log.info("Processing File-Based Catalog image")
```

Comment:
Is there any reason behind adding this check and log here? We can just add this log on line 756 directly as well no?

Reply:
But then you would have this info after you started working with the FBC image, right? (e.g. `_get_present_bundles`)
My intention was to have this info before any operation done on the image.